hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java

```diff
@@ -256,9 +256,9 @@ public <T extends FileSystem & Renewable> void removeRenewAction(
       try {
         action.cancel();
       } catch (InterruptedException ie) {
-        LOG.error("Interrupted while canceling token for " + fs.getUri()
+        LOG.error("Interrupted while canceling token for {} filesystem." + fs.getUri()
```

Comment:
needs to move the fs.geturi to a param, not string concat

Reply:
Thanks for the help reviewing the code, I will fix it.